image-prod.yaml

```diff
@@ -0,0 +1,55 @@
+- schema_version: 1
+  name: "builder"
+  description: "Golang builder image"
+  version: ""
+  from: "registry.redhat.io/ubi8/go-toolset:1.17.7"
+  modules:
+    repositories:
+      - name: builder
+        path: modules/builder-prod
+    install:
+      - name: builder
+
+  osbs:
+    configuration:
+      container:
+        image_build_method: imagebuilder
+        remote_source:
+          repo: https://github.com/kiegroup/kie-cloud-operator
+          ref: 7b5d7503589e7c19bd76a6fa5a33bf313ee685ba
+          pkg_managers:
+            - gomod
+        platforms:
+          only:
+            - x86_64
+
+- schema_version: 1
+  name: "rhpam-7/rhpam-rhel8-operator"
+  description: "Red Hat Business Automation Operator"
+  version: ""
+  from: "registry.redhat.io/ubi8-minimal"
+  labels:
+    - name: maintainer
+      value: bsig-cloud@redhat.com
+    - name: com.redhat.component
+      value: rhpam-7-rhel8-operator-container
+    - name: io.k8s.description
+      value: Operator for deploying RHPAM & RHDM
+    - name: io.k8s.display-name
+      value: Red Hat Business Automation Operator
+    - name: io.openshift.tags
+      value: rhpam,rhdm,operator
+  modules:
+    repositories:
+      - name: runtime
+        path: modules/runtime
```

Comment:
where is this module?

I would expect to see something similar to:

```
  artifacts:
    - path: <path>
      image:builder
      dest: <dest>
```

Reply:
@spolti IIUC the artifacts are specified on _modules/runtime/module.yaml_ file.

Reply:
ok, but from where this runtime module is coming from?

Reply:
Not sure, I got this file as example from image.yaml.

Reply:
which image.yaml?

Reply:
I'd be surprised if the build don't fail, since CeKit will try to look for an non existent module.

Reply:
@spolti isn't that modules/runtimes this one https://github.com/kiegroup/kie-cloud-operator/tree/release-v7.13.x/modules/runtime ? Sorry if I'm missing something.

Actually we had this same block in the original `image.yaml` file, see https://github.com/kiegroup/kie-cloud-operator/blob/release-v7.13.x/image.yaml#L36-L39, and we just based this new yaml on it.

Reply:
you guys are correct, sry, my bad :)